crates/rpc/ipc/src/server/mod.rs

```diff
@@ -411,78 +447,42 @@ where
 async fn spawn_connection<S, T>(
```

Comment:
this no longer spawns a connections, rename to process

Reply:
Done.